src/plugins/messenger/MessengerPreview/components/MessengerImage/components/ImagePreview.tsx

```diff
@@ -0,0 +1,85 @@
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import { getBackgroundImage } from '../../../lib/css';
+
+export const getImagePreview = ({ React, styled }: MessagePluginFactoryProps) => {
+
+    const templateStyle = ({ template = 'media' }) => ({
+        media: {
+            paddingTop: "75%"
+        },
+        list: {
+            paddingTop: "50%"
+        },
+        generic: {
+            paddingTop: "50%",
+            display: "table",
+            width: "100%"
+        }
+    })[template];
+
+    const FixedImage = styled.div({
+        backgroundSize: "cover",
+        backgroundPosition: "center center",
+        cursor: 'pointer',
+        "&:hover": {
+            opacity: '.8'
+        },
+        "&:focus": {
+            opacity: '.8',
+            outline: 'none'
+        },
+    }, templateStyle);
+
+    const FlexImage = styled.img({
+        display: 'block',
+        width: '100%',
+        cursor: 'pointer',
+        "&:hover": {
+            opacity: '.8'
+        },
+        "&:focus": {
+            opacity: '.8',
+            outline: 'none'
+        },
```

Comment:
This won't work. The focus style should be added to the div wrapping this with tabIndex=0